workshops/models.py

```diff
@@ -2759,6 +2759,9 @@ class WorkshopRequest(AssignmentMixin, StateMixin, CreatedUpdatedMixin,
         "<a href='http://www.datacarpentry.org/lessons/'>"
         "Data Carpentry lessons page</a>"
     )
+    LC_LESSONS_LINK = (
+        "<a href='https://librarycarpentry.org/'>"
+        "Library Carpentry</a>"
```

Comment:
```suggestion
        "Library Carpentry</a>"
    )
```

This fixes syntax error.